tools/bin/mbedtls-rewrite-branch-style

```diff
@@ -385,6 +392,33 @@ class CodeStyleBranchRewriter(MbedTLSBranchRewriter):
             ok = False
         return ok
 
+    def maybe_fetch(self, ref: str) -> None:
```

Comment:
Minor: this name suggests that in some (unclear) circumstances there will be no fetch, while I believe it just wont happen if an error occurs.

Reply:
There's no fetch if the ref already exists. This was missing from the documentation, I've fixed that.